server/tests/api_v4/anonymous/test_organization_bootstrap.py

```diff
@@ -1,44 +1,87 @@
 # Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-# TODO: test organization_already_bootstrapped is not returned if a invalid token is provided
+from collections.abc import Awaitable
+from dataclasses import dataclass
+from typing import Protocol
+
 import pytest
 
 from parsec._parsec import BootstrapToken, DateTime, OrganizationID, anonymous_cmds
 from parsec._parsec import testbed as tb
+from parsec.ballpark import BALLPARK_CLIENT_EARLY_OFFSET, BALLPARK_CLIENT_LATE_OFFSET
 from tests.common import AnonymousRpcClient, AsyncClient, Backend, TestbedBackend
 
 
-@pytest.mark.parametrize("kind", ("standard", "sequestered", "spontaneous"))
-async def test_anonymous_organization_bootstrap_ok(
-    kind: str,
+@pytest.fixture
+def organization_id() -> OrganizationID:
+    return OrganizationID("NewOrg")
+
+
+@pytest.fixture
+def anonymous_client(client: AsyncClient, organization_id: OrganizationID) -> AnonymousRpcClient:
+    return AnonymousRpcClient(client, organization_id)
+
+
+@dataclass
+class BackendPreConfig:
+    spontaneous: bool
+    sequestered: bool
+
+
+@dataclass
+class BackendConfig:
+    spontaneous: bool
+    sequestered: bool
+    bootstrap_token: BootstrapToken | None
+
+
+class ConfigureBackend(Protocol):
+    def __call__(self, organization_id: OrganizationID) -> Awaitable[BackendConfig]:
+        ...
+
+
+@pytest.fixture(
+    params=[
+        pytest.param(BackendPreConfig(False, False), id="standard"),
```

Comment:
would be more readable to pass the parameter name here, e.g. `BackendPreConfig(spontaneous=False, sequestered=False)`

Reply:
Yes, I've configured by IDE to show those value, but on Github they aren't present